.circleci/config.yml

```diff
@@ -83,7 +88,7 @@ jobs:
           command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.27.1/wiremock-standalone-2.27.1.jar
       - run:
           name: Run wiremock
-          command: java -jar wiremock.jar --port 9191
+          command: java -jar wiremock.jar --port 9091
```

Comment:
moved wiremock over to 9091 as its cumbersome having to run a different version on a different port to the other services